targets/tizenrt-artik05x/apps/jerryscript/jerry_main.c

```diff
@@ -0,0 +1,326 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <setjmp.h>
+
+#include "jerryscript.h"
+#include "jerryscript-port.h"
+#include "jmem.h"
+
+#include <apps/shell/tash.h> // To register tash command
+
+/**
+ * Maximum command line arguments number.
+ */
+#define JERRY_MAX_COMMAND_LINE_ARGS (16)
+
+/**
+ * Standalone Jerry exit codes.
+ */
+#define JERRY_STANDALONE_EXIT_CODE_OK   (0)
+#define JERRY_STANDALONE_EXIT_CODE_FAIL (1)
+
+/**
+ * Print usage and available options
+ */
+static void
+print_help (char *name)
+{
+  jerry_port_console ("Usage: %s [OPTION]... [FILE]...\n"
```

Comment:
I tried to remove `jerry_port_console`. However, `jerry-core/ecma/builtin-objects/ecma-builtin-global.c` uses `jerry_port_console`. Perhaps I can remove `jerry_port_console` after #1749 is merged.